sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -231,3 +231,29 @@ CREATE TEMPORARY VIEW fmtTable(fmtField) AS SELECT * FROM VALUES ('invalidFormat
 SELECT to_binary('abc', fmtField) FROM fmtTable;
 -- Clean up
 DROP VIEW IF EXISTS fmtTable;
+-- luhn_check
+select luhn_check('4111111111111111');
+select luhn_check('5500000000000004');
+select luhn_check('340000000000009');
+select luhn_check('6011000000000004');
+select luhn_check('378282246310005');
+select luhn_check('6011000990139424');
+select luhn_check('1234567890');
+select luhn_check('4111111111111');
+select luhn_check('4111111111111112');
+select luhn_check('371449635398431');
+select luhn_check('371449635398432');
+select luhn_check('30569309025904');
+select luhn_check('30569309025905');
+select luhn_check('6011111111111117');
+select luhn_check('6011111111111118');
+select luhn_check('3530111333300000');
+select luhn_check('3530111333300001');
+select luhn_check('5105105105105100');
+select luhn_check('5105105105105106');
+select luhn_check('510B105105105106');
+select luhn_check('E105105105105106');
```

Comment:
also add some negative test cases where the input is the wrong type? and some positive tests where the input is an attribute reference instead of a constant string?

Reply:
Added additional test cases